spec/fixtures/mock_lcnaf.yml

```diff
@@ -0,0 +1 @@
+"New York (N.Y.). Administration for Children's Services":"http://id.loc.gov/authorities/names/n2001099999"
```

Comment:
This is necessary because if we try to mock out File#open we would need to modify the HeaderLookup#initialize function which would ultimately be, as the kids say, a hot mess.